src/main/resources/static/app/service/LoginService.js

```diff
@@ -0,0 +1,25 @@
+var userService = angular.module('greatStartApp')
+    .factory('loginService', function ($rootScope, $http) {
```

Comment:
Name of factories usually starts from uppercase letter - the same as for Java classes